vitrina/users/models.py

```diff
@@ -114,6 +114,16 @@ def is_gov_organization_manager(self) -> bool:
             open_data_representative=False,
         ).exists()
 
+    @property
+    def is_gov_organization_information_system_manager(self) -> bool:
+        gov_org_content_type = self.organization_content_type
```

Comment:
Nitpicky suggestion: The naming suggests that organizations that are of type `gov`, have a different content_type than other organizations, therefore I suggest:

```suggestion
        org_content_type = self.organization_content_type
```

Or even better, just use the `self.organization_content_type` inside the filter, we are not re-using the `gov_org_content_type` more than once, no need to put it into a separate variable